pkgs/applications/display-managers/sddm/sddm-catppuccin.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchFromGitHub, flavor ? "mocha" }:
+stdenv.mkDerivation {
+  pname = "sddm-catppuccin-theme";
+  version = "0.0.0b0";
```

```suggestion
{ stdenv, lib, fetchFromGitHub, flavor ? "mocha" }:

stdenv.mkDerivation {
  pname = "sddm-catppuccin-theme";
  version = "unstable-XXXX-XX-XX";
```
XXXX-XX-XX should be the date of the commit in iso format

---

resolved in [517edfb](https://github.com/NixOS/nixpkgs/pull/240990/commits/517edfbaeaad7c040ffa212334e4584816f2a2d1)